app/controllers/ModelScorer.scala

```diff
@@ -11,10 +11,16 @@ object ModelScorer {
   Logger.getLogger("org").setLevel(Level.OFF)
   Logger.getLogger("akka").setLevel(Level.OFF)
 
+  val conf = new org.apache.spark.SparkConf()
+    .setSparkHome("/Users/ykhodorkovsky/spark-2.2.0-bin-hadoop2.7")
+    .setMaster("spark://ip-192-168-56-190.ec2.internal:7077")
```

Comment:
You can start a local spark master with `SPARK_HOME/sbin/start-all.sh`. Then navigate to the master UI `http://localhost:8080`. There you'll have all those values (master address etc.)